zeroex/order.go

```diff
@@ -142,7 +142,7 @@ type OrderEvent struct {
 }
 
 type orderEventJSON struct {
-	Timestamp                time.Time            `json:"timestamp"`
+	Timestamp                string               `json:"timestamp"`
```

Comment:
I don't think this change is necessary. `encoding/json` supports struct fields of type `time.Time` out of the box. See this playground example: https://play.golang.org/p/FccYSYAXZvV

Reply:
The reason why I did this is the same as the other PR. We cannot convert `time.Time` to a `js.Value`, so at least in that case, we have to make a conversion like this. I thought it made sense to keep the Javascript and JSON values consistent, but I'm also happy to change if you disagree.